src/functional/iterator/transforms/inline_lambdas.py

```diff
@@ -31,7 +31,7 @@ def visit_Lambda(self, node: ir.Lambda, *, active_refs: set[str]):
 
 @dataclasses.dataclass
 class InlineLambdas(NodeTranslator):
-    """Inline lambda calls by substituting every arguments by its value."""
+    """Inline lambda calls by substituting every argument by its value."""
```

Comment:
I guess this is an unrelated change...

Reply:
Yes, still had allowed it to sneak in :-)